package.json

```diff
@@ -106,7 +106,8 @@
     "jquery-ui": "https://jqueryui.com/resources/download/jquery-ui-1.12.1.zip",
     "jquery-ui-themes": "https://jqueryui.com/resources/download/jquery-ui-themes-1.12.1.zip",
     "literallycanvas": "https://github.com/literallycanvas/literallycanvas/archive/v0.4.14.tar.gz",
-    "react": "https://github.com/facebook/react/releases/download/v15.1.0/react-15.1.0.zip"
+    "react": "https://github.com/facebook/react/releases/download/v15.1.0/react-15.1.0.zip",
+    "lforms": "https://clinicaltables.nlm.nih.gov/lforms-versions/lforms-30.4.1.zip"
```

Comment:
bummer that the project is not keeping up with npm (looks like only up to version 29.3.1 there), since goal at some point is to whittle these things away. But we can't do anything about that :)

Reply:
npm lforms is for node or the use of requires which project doesn't recm to use.